Index: DataSourcesModule/src/main/java/mariusz/ambroziak/kassistant/hibernate/dao/ProductDAO.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package mariusz.ambroziak.kassistant.hibernate.dao;\n\nimport java.util.List;\n\nimport mariusz.ambroziak.kassistant.hibernate.model.ProductData;\n\n\npublic interface ProductDAO {\n\tpublic List<ProductData> list();\n//\tpublic List<ProductData> getProduktsById(String id);\n\tpublic void saveProduct(ProductData p);\n//\n//\t\n\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- DataSourcesModule/src/main/java/mariusz/ambroziak/kassistant/hibernate/dao/ProductDAO.java	(revision 61bc9730c7f6f7136d12d0de3b942a87182360be)
+++ DataSourcesModule/src/main/java/mariusz/ambroziak/kassistant/hibernate/dao/ProductDAO.java	(date 1586196402958)
@@ -7,7 +7,9 @@
 
 public interface ProductDAO {
 	public List<ProductData> list();
-//	public List<ProductData> getProduktsById(String id);
+	public List<ProductData> getProduktsById(String id);
+	public List<ProductData> getProduktsByFullUrl(String url);
+
 	public void saveProduct(ProductData p);
 //
 //	
